setup.py

```diff
@@ -8,7 +8,7 @@
 
 setup(
     name='xbundle',
-    version="0.3.0",
+    version="0.3.1",
```

Comment:
I'd like a 2nd opinion on the version number: @bdero is this XBundle v0.3.1 or XBundle 0.4.0 ?